packages/edit-post/src/components/visual-editor/index.js

```diff
@@ -217,36 +217,28 @@ export default function VisualEditor( { styles } ) {
 									layout={ defaultLayout }
 								/>
 							) }
-							<AnimatePresence>
-								<motion.div
```

Comment:
I wonder if we had style targeting that div, specifically something about making it full height

Reply:
I'm not sure what the best way is to search for this. The wrapping container, `.edit-post-visual-editor__content-area`, only appears once, and does not appear to target any child divs.

It worked well in my testing, but I'll keep an eye on this and follow up on anything that needs it!